### ISSUE TITLE
Pip 20.0.0+ compatibility

### ISSUE THREAD

**Issue by Korijn** [almarklein/wgpu-py#46]
Unfortunately our wheels are no longer accepted as of pip 20.0.0: https://github.com/pypa/pip/pull/7354

It's considered an invalid platform tag :(

**Comment by Korijn** [comment]
I guess we should open an issue upstream and see if they want to consider our use case. If not, we'll have to provide separate wheels for all versions of python we want to support...

**Comment by almarklein** [comment]
WTF - there should be many projects that have the same use-case. Any project that brings binaries that are not Python modules, right? Thinking of pyglfw to name one ...

**Comment by Korijn** [comment]
It seems like that, yes...

**Comment by Korijn** [comment]
I just checked with [glfw](https://pypi.org/project/glfw/#files) since they have the same wheel platform tag convention we do, to confirm, and sure enough, you can see that pip will select the source distribution over the wheels that are available. If I force to install the wheel that _should_ be fine on my system, it errors out:

```
$ pipenv install pip>=20.0.0
Creating a virtualenv for this project…
Pipfile: C:\Users\korij\dev\test\Pipfile
Using c:\users\korij\appdata\local\programs\python\python36\python.exe (3.6.4) to create virtualenv…
[  ==] Creating virtual environment...Already using interpreter c:\users\korij\appdata\local\programs\python\python36\python.exe
Using base prefix 'c:\\users\\korij\\appdata\\local\\programs\\python\\python36'
New python executable in C:\Users\korij\.virtualenvs\test-SUagSMsE\Scripts\python.exe
Installing setuptools, pip, wheel...done.

Successfully created virtual environment!
Virtualenv location: C:\Users\korij\.virtualenvs\test-SUagSMsE

$ pipenv run pip --version
pip 20.0.1 from c:\users\korij\.virtualenvs\test-suagsmse\lib\site-packages\pip (python 3.6)

$ pipenv run pip install glfw==1.10.1
Collecting glfw==1.10.1
  Downloading glfw-1.10.1.tar.gz (25 kB)
Building wheels for collected packages: glfw
  Building wheel for glfw (setup.py) ... done
  Created wheel for glfw: filename=glfw-1.10.1-py3-none-any.whl size=25667 sha256=29c3f9e8a609b0884a418d13ccbc7b2a9feae3e76a71276111e0673167f68487
  Stored in directory: c:\users\korij\appdata\local\pip\cache\wheels\a0\b4\23\11682f5036ba8a4d9711066c08be9325530f5330930b71b513
Successfully built glfw
Installing collected packages: glfw
Successfully installed glfw-1.10.1

$ pipenv run pip uninstall glfw
Found existing installation: glfw 1.10.1
Uninstalling glfw-1.10.1:
  Would remove:
    c:\users\korij\.virtualenvs\test-suagsmse\lib\site-packages\glfw-1.10.1.dist-info\*
    c:\users\korij\.virtualenvs\test-suagsmse\lib\site-packages\glfw\*
Proceed (y/n)? y
  Successfully uninstalled glfw-1.10.1

$ pipenv run pip install https://files.pythonhosted.org/packages/2a/f2/2d6381ae04caf456b63894278d875f3b6723aad895f0839c39e11e84b0d0/glfw-1.10.1-py2.py3-none-win_amd64.whl
ERROR: glfw-1.10.1-py2.py3-none-win_amd64.whl is not a supported wheel on this platform.
```

**Comment by Korijn** [comment]
Opened an issue upstream: https://github.com/pypa/pip/issues/7636 please let me know if I was unclear about anything.

**Comment by Korijn** [comment]
The root cause was apparently an accidentally reused iterator, see https://github.com/pypa/packaging/pull/258

Just gotta wait for the release cycle to deliver the goods

**Comment by Korijn** [comment]
Closed by #50
